### ISSUE TITLE
keybase.kbfs not starting on Mac OS X

### ISSUE THREAD

**Issue by bgschaid** [keybase/client#3576]
I yesterday download the binary release to play around with the filesystem. But it is not working. I keep getting these two messages in the `system.log`:

    Jul 22 10:30:43 bgs-cool-greybook com.apple.xpc.launchd[1] (keybase.kbfs[9485]): Service exited with abnormal code: 2
    Jul 22 10:30:43 bgs-cool-greybook com.apple.xpc.launchd[1] (keybase.kbfs): Service only ran for 0 seconds. Pushing respawn out by 10 seconds.

Don't know if this is the problem: I had two previous installations of keybase: one through `npm`and one through `MacPorts`. Both of them were uninstalled. The Folder `/keybase` already exists and dates from February (when I played with KeyBase). Don't remember if I created it or it was created by some program.  Rights to that folder are

    drwxr-xr-x    2 root      wheel     68 Feb  9 20:36 keybase

my log id: cf58777ee4cc226dde8e981c

**Comment by maxtaco** [comment]
Looking at your logs:

```
891 2016-07-22T10:29:41.448361 ▶ [DEBU kbfs main.go:107] 002 Mounting: /keybase
892 2016-07-22T10:29:41.448399 ▶ [DEBU keybase login_state.go:882] 003 LoginState: Running request loop
893 2016/07/22 10:29:41 mount helper error: mount_kbfuse: failed to mount /keybase@/dev/kbfuse0: Operation not permitted
894 2016/07/22 10:29:41 mount helper error: mount_kbfuse: failed to mount /keybase@/dev/kbfuse0: Operation not permitted
```

@gabriel any ideas on this?

**Comment by bgschaid** [comment]
I think I "fixed" it:
I removed the `/keybase`-folder and then killed every process with "keybase" in the name (a bit brutal). Then the Application reopened and I was asked to enter my passphrase and the folders `/keybase/public` and `/keybase/private` were there and now it seems to work.
My guess what happend is: when playing around in February the `/keybase`folder was created (possibly by me). Now when I downloaded the app and installed it the initalisation routine said "cool. I was already here because the `/keybase`folder is here. Let the mounting begin" and the part in charge of mounting said "Na. Subfolders are missing. I'll try again in 10 seconds"

**Comment by gabriel** [comment]
Yeah, the installer isn't smart enough to detect and fix a invalid existing placeholder dir for /keybase. It is something we'll get to though. Thanks for the feedback.

**Comment by gabriel** [comment]
Tracking this internally in our JIRA as https://keybase.atlassian.net/browse/DESKTOP-1554
